outputs.tf

```diff
@@ -20,11 +20,10 @@ output "agent_proxy_container" {
             }
         } : null)
         environment = [
-            {"name": "AEMBIT_AGENT_CONTROLLER", "value": "http://${aws_service_discovery_service.agent-controller.name}.${aws_service_discovery_private_dns_namespace.agent-controller.name}:80"},
-            {"name": "TRUSTED_CA_CERTS", "value": var.aembit_trusted_ca_certs },
+            {"name": "AEMBIT_AGENT_CONTROLLER", "value": "https://${aws_service_discovery_service.agent-controller.name}.${aws_service_discovery_private_dns_namespace.agent-controller.name}:443"},
+            {"name": "TRUSTED_CA_CERTS", "value": data.http.trusted_ca_cert.response_body_base64 },

```

Comment:
Unfortunately, this is not enough. 

We want to have both
a) Allow to pass additional trusted root ca (via var.aembit_trusted_ca_certs). This is for the case if server workload uses self-signed cert (so AP can connect)
b) Add Tenant Root CA

BTW. This is the reason for that base64 deconding, appening, encoding that I was adding to JIRA.